core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,201 @@
+
+                    options.id = subscriber.get('id');
+                    return models.Subscriber.edit(subscriber.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Subscribers!');
+                });
+            });
+        },
+        function () {
+            return models.Settings.forge().fetchAll().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Settings found');
+                    return;
+                }
+
+                return Promise.mapSeries(result.models, function mapper(settings) {
+                    if (settings.get('updated_at')) {
+                        settings.set('updated_at', _private.addOffset(settings.get('updated_at')));
+                    }
+
+                    settings.set('created_at', _private.addOffset(settings.get('created_at')));
+
+                    options.id = settings.get('id');
+                    return models.Settings.edit(settings.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Settings!');
+                });
+            });
+        },
+        function () {
+            return Promise.all(['Roles', 'Permissions', 'Tags', 'Apps', 'AppSettings', 'AppFields', 'Clients'].map(function (model) {
+                return models[model].forge().fetch().then(function (result) {
+                    if (result.models.length === 0) {
+                        logger.warn(messagePrefix + 'No {model} found'.replace('{model}', model));
+                        return;
+                    }
+
+                    return Promise.mapSeries(result.models, function mapper(object) {
+                        object.set('created_at', _private.addOffset(object.get('created_at')));
+
+                        if (object.get('updated_at')) {
+                            object.set('updated_at', _private.addOffset(object.get('updated_at')));
+                        }
+
+                        options.id = object.get('id');
+                        return models[model.slice(0, -1)].edit(object.toJSON(), options);
+                    }).then(function () {
+                        logger.info(messagePrefix + 'Updated datetime fields for {model}!'.replace('{model}', model));
+                    });
+                });
+            }));
+        },
+        function () {
+            settingsMigrations[settingsKey] = moment().format();
+            return models.Settings.edit({
+                key: 'migrations',
+                value: JSON.stringify(settingsMigrations)
+            }, {context: {internal: true}});
+        }]
+    ).catch(function (err) {
+        if (err.message === 'skip') {
+            logger.info(messagePrefix + 'Your databases uses UTC datetimes, skip!');
```

Comment:
This should be a `logger.warn` message, we use `info` for success and `warn` for "skipped" even if the migration is skipped for a good reason, it just makes it easer to read what did/didn't happen